src/modules/solution-explorer/solution-explorer-panel/solution-explorer-panel.html

```diff
@@ -36,4 +44,26 @@
 
     </div>
   </div>
+
+  <!-- Open Remote Solution Modal -->
+  <div show.bind="showOpenRemoteSolutionModal">
+    <div class="modal show show-modal" tabindex="-1" role="dialog" id="openRemoteSolutionModal">
+      <div class="modal-dialog open-remote-solution-modal-dialog" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title">Open Remote Solution</h4>
+          </div>
+          <div class="modal-body">
+            URI:
```

Comment:
```suggestion
            Location:
```

This will most certainly not be a URI.